www/md_base/src/app/builds/builder/forcedialog/forcedialog.controller.coffee

```diff
@@ -16,26 +16,24 @@ class ForceDialog extends Controller
         @$mdDialog.hide()
 
     forceBuildFail: (error) ->
-        if error.code == -32602
+        if error.code == -32602 # Non-networking error
```

Comment:
put it at `common.constant.coffee` ? As this constant won't be used at other place, I think a comment might be ok enough :)